### ISSUE TITLE
Doubt about JCB binary message assembly

### ISSUE THREAD

**Issue by Juniornewxt** [moov-io/iso8583#136]
Hello friends, I have a problem here, with messages from JCB, I would like to specify a message in the following format below?

Bit 33
Length: 1 binary byte, then up to 11 digits (maximum of 7 bytes)
Format: LLVAR binary, numeric 4-bit BCD (unsigned, packed)

If yes, can you send me an example?
I've tried it in several ways without success.

Tks friends.

**Comment by alovak** [comment]
I think it should look something like this when you define the spec:

```
		33: field.NewString(&field.Spec{
			Length:      11,
			Description: "Forwarding Institution Identification Code",
			Enc:         encoding.BCD,
			Pref:        prefix.BCD.LL,
		}),
```

**Comment by alovak** [comment]
Hey @Juniornewxt ! Was it solved by #132 ? Asking to see if I can close the issue or it needs some work.

**Comment by Juniornewxt** [comment]
you can close it

tks
